tokio-util/src/codec/framed_write.rs

```diff
@@ -18,8 +18,15 @@ pin_project! {
     /// For examples of how to use `FramedWrite` with a codec, see the
     /// examples on the [`codec`] module.
     ///
+    /// # Cancellation safety
+    ///
+    /// * [`futures_util::sink::SinkExt::send`]: if send is used as the event in a
+    /// `tokio::select!` statement and some other branch completes first, then it is
+    /// guaranteed that the message was not sent, but the message itself is lost.
+    ///
     /// [`Sink`]: futures_sink::Sink
     /// [`codec`]: crate::codec
+    /// [`futures_util::sink::SinkExt::send`]: https://docs.rs/futures-util/latest/futures_util/sink/trait.SinkExt.html#method.send
```

Comment:
Does this not work?
```suggestion
    /// [`futures_util::sink::SinkExt::send`]: futures_util::sink::SinkExt::send
```
If it does, please change it. Otherwise, it's fine.